testsupport/user_assertions.go

```diff
@@ -102,7 +102,23 @@ func VerifyResourcesProvisionedForSignup(t *testing.T, awaitilities wait.Awaitil
 		}
 	}
 
-	tiers.VerifyNsTemplateSet(t, hostAwait, memberAwait, userAccount, tierName)
+	tier, err := hostAwait.WaitForNSTemplateTier(mur.Spec.TierName)
+	require.NoError(t, err)
+	hash, err := testtier.ComputeTemplateRefsHash(tier) // we can assume the JSON marshalling will always work
+	require.NoError(t, err)
+
+	space, err := hostAwait.WaitForSpace(mur.Name,
+		wait.UntilSpaceHasTier(mur.Spec.TierName),
+		wait.UntilSpaceHasLabelWithValue(toolchainv1alpha1.SpaceCreatorLabelKey, userSignup.Name),
+		wait.UntilSpaceHasLabelWithValue(fmt.Sprintf("toolchain.dev.openshift.com/%s-tier-hash", mur.Spec.TierName), hash),
+		wait.UntilSpaceHasConditions(Provisioned()),
+		wait.UntilSpaceHasStateLabel(toolchainv1alpha1.SpaceStateLabelValueClusterAssigned),
+		wait.UntilSpaceHasStatusTargetCluster(mur.Spec.UserAccounts[0].TargetCluster))
+	require.NoError(t, err)
+
+	VerifySpaceBinding(t, hostAwait, mur.Name, space.Name, "admin")
+
+	tiers.VerifyNsTemplateSet(t, hostAwait, memberAwait, space, tierName)
```

Comment:
it can be done in separate PR, but is there any difference between your code that the one that is executed in `VerifyResourcesProvisionedForSpace`?

Reply:
Noted, will handle in a separate PR.